### PR TITLE
Admin tour + help text updates (SH-430/SH-445)

### PR DESCRIPTION
**Pull request by gurch101**
* Add admin walkthrough
* Show help text as a popover

Refs SH-430/SH-445

### CODE REVIEW

**Comment by Pikkupomo** [comment]
The tour is missing Titles in all of the boxes. IMO they should be present, now the tour boxes seem too "naked".

**Comment by Pikkupomo** [comment]
When going through the menu, it would be nice to have all other menu entries (but the one currently described) somehow blurred.

File: shuup/admin/static_src/base/js/tour.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        }, {
+            text: [
+                gettext("Find anything, and I mean anything that is inside your store. This includes settings, products, users, and addons.")
+            ],
+            attachTo: "#site-search bottom"
+        }, {
+            text: [
+                gettext("Peep your shop and all the cool changes you’ve made.")
+            ],
+            attachTo: ".shop-btn left"
```
Comment (gurch101) [inline shuup/admin/static_src/base/js/tour.js]: yea seems to be an issue in the third party lib. Anything near the edges of the screen is missing arrows.

File: shuup/admin/static_src/base/js/tour.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
+/**
+ * This file is part of Shuup.
+ *
+ * Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+ *
+ * This source code is licensed under the AGPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+(function($) {
```
Comment (gurch101) [inline shuup/admin/static_src/base/js/tour.js]: This code is used for all tours. When "re-running" a tour, we run the tour for the page + the chrome.

**Pull request merged**